package.json

```diff
@@ -10,7 +10,7 @@
     "lint-autofix": "eslint . --ext .ts,.tsx --fix",
     "format-check": "prettier --check src",
     "format-write": "prettier --write src",
-    "start": "nodemon --exec ts-node --esm ./src/index.ts",
+    "start": "node --watch --no-warnings --loader ts-node/esm ./src/index.ts",
```

Comment:
why `--no-warnings`? (not sure what warnings these are, does this mean we risk overlooking them?)

Reply:
It's the only way to turn off warnings for experimental features like `--watch`. We can remove it, but it's a bit noisy when it starts up

Reply:
OK to keep it like this if you don't think there's too much risk to overlook new relevant warnings